module/input/SensorFilter/src/Kinematics.hpp

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
I'd recommend making a subfolder to collect all these cpps, like with extension/director

Reply:
oh I just noticed they are hpps. These are not headers, just implementation for the class functions, name them with the `cpp` extension and remove the macroguards.